core/src/main/resources/application.conf

```diff
@@ -22,6 +22,188 @@ raphtory {
       size                  = ${?RAPHTORY_RETENTION_SIZE}
 
     }
+    topics {
```

Comment:
The topics are defined by and `id` and a `subTopic`. We could define them by a `group` and `id` and a `subTopic` instead. This way we would be able to manage the configuration per topic group to simplify `application.conf`. It would be similar to the previous sorting but decoupled from the components.